hack/test/patches/image-cache.yaml

```diff
@@ -2,20 +2,6 @@ machine:
   features:
     imageCache:
       localEnabled: true
-  registries:
```

Comment:
was this intentional, this was added to make sure we never reach out to the internet

Reply:
oops, I should revert